test/index.html

```diff
@@ -9,6 +9,9 @@
 <body>
   <div id="qunit"></div>
   <script src="../node_modules/qunitjs/qunit/qunit.js"></script>
+  <script>
+    QUnit.config.reorder = false;
```

Comment:
I don't think so. I think setting `reorder` to `false` is really used to mask bad cleanup in tests. I think @gkatsev may have cleaned that up in another PR as well.